frontend/src/pages/BYONImages/tableData.tsx

```diff
@@ -35,6 +35,16 @@ export const columns: SortableData<BYONImage>[] = [
       popover: 'Accelerators are used to speed up the execution of workbenches.',
     },
   },
+  {
+    field: 'recommendedHardwareProfiles',
+    label: 'Recommended hardware profiles',
+    sortable: (a, b) =>
```

Comment:
nit pick -- possible to use more contextual variable names like (left, right) or (top, bottom) etc?

Reply:
This is a sort helper function, `a` and `b` doesn't have any meaning here, it would be confusing if we change it to left/right or top/bottom because when you look the function below, you will wonder "what's the meaning of left.recommendedAcceleratorIdentifiers.length - right.recommendedAcceleratorIdentifiers.length?". What we want to do here is only to use the first one to minus second one to make the sortable function working as expected.